src/spec-converter.js

```diff
@@ -132,7 +131,7 @@ export class SpecConverter {
     ruleAssignStructure(srcSpec, gplSpec) {
 
         var walkStructure = (srcUnit) => {
-            var gplRoot = utils.clone(_.omit(srcUnit, 'unit'));
+            var gplRoot = Object.assign({}, utils.omit(srcUnit, 'unit'));
```

Comment:
The utils.clone made deep clone here. I'm not sure it is equivalent refactoring.